get.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+: '
+    Copyright (C) 2020 IBM Corporation
+
+    Rafael Sene <rpsene@br.ibm.com> - Initial implementation.
+
+    This is a helper script that eases the dependency setup and 
+    configuration for the pvsadm tool. 
+'
+
+# Trap ctrl-c and call ctrl_c()
+trap ctrl_c INT
+
+function ctrl_c() {
+    echo "Bye!"
+}
+
+function identify_os() {
+
+    local OS="$(uname -s)"
+
+    case "${OS}" in
+        Linux*)     DISTRO=linux;;
+        Darwin*)    DISTRO=darwin;;
+        Catalina*)  DISTRO=darwin;;
+        *)          DISTRO="UNKNOWN:${OS}"
+    esac
+
+    ARCH=$(uname -m)
+
+    if [ "$ARCH" == "amd64" ] || [ "$ARCH" == "x86_64" ]; then
+        ARCH=amd64
+    fi
+
+    export $ARCH
+    export $DISTRO
+}
+
+function check_connectivity() {
+    
+    curl --output /dev/null --silent --head --fail http://ibm.com
```

Comment:
how about pinging github.com instead of ibm.com? as we are pulling the code from github.com